helm-charts/0.26.2/charts/sentinel/templates/Role.yaml

```diff
@@ -8,9 +8,15 @@
 # >/'  SPDX-License-Identifier: BSD-2-Clause
 # */
 
-apiVersion: v1
-kind: ServiceAccount
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
 metadata:
-  name: example
-  namespace: default
-automountServiceAccountToken: false
+  name: vsecm-sentinel-secret-reader
+  namespace: {{ .Values.global.vsecm.namespace }}
+rules:
+  - apiGroups: [""]
+    resources: ["secrets"]
+    verbs: ["get", "list", "watch"]
+    resourceNames: ["vsecm-sentinel-init-secret"]
```

Comment:
After defining the service account to explicitly list the secrets it allows to mount, we needed to create a role and rolebinding to support that.

This is a couple of extra annotations, but better security too.